src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -16,7 +16,29 @@ const AuctionsApp: React.FC<AuctionsAppProps> = ({ children }) => {
   return (
     <AppContainer>
       <AuctionsMeta />
-
+      <Box mt={3} ml={4}>
```

Comment:
In the mobile screenshot It looks like the left hand margin doesn't quite line up with the overall container and tab bar, like it does on desktop. 

<img width="48" alt="Screen Shot 2021-02-25 at 9 09 43 PM" src="https://user-images.githubusercontent.com/236943/109257833-d31ef100-77ad-11eb-9462-d54873f99ccf.png">


But this is an easy fix! 

Palette has a cool feature where we can do responsive layouts really easily by passing an array to our props; each "slot" in the array represents a breakpoint. So like:

```tsx
ml={[xs, sm, md, lg, xl]}
```

These values correspond to the breakpoints in our theme file here: https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx#L23-L34

So if you want, say, no margin-left on mobile but a margin on desktop you can do 

```tsx
<Box ml={[0, 4]}>
```

And from left to right, it will apply values to the right so you don't need to fill in every slot, as a form of shorthand; meaning, these two are identical:

```tsx
<Box ml={[0, 4, 4, 4 ,4]]>
<Box ml={[0, 4]]>
```

Reply:
Thank you @damassi ! I was looking for a way to style mobile specific view, and I added in your suggestions.